quickwit/Makefile

```diff
@@ -21,6 +21,11 @@ fix:
 	@echo "Running cargo clippy --fix"
 	@cargo clippy --workspace --all-features --tests --fix --allow-dirty --allow-staged
 	@$(MAKE) fmt
+	@$(MAKE) unused-deps
+
+unused-deps:
+	@echo "Checking for unused dependencies"
+	@cargo machete
```

Comment:
Check whether cargo machete is installed and display the command to install if not the same way it's done for the nightly toolchain in fmt

Reply:
Ok. Let's not run with `--fix` if ti removes false positives as well then.